race_control_http.go

```diff
@@ -392,3 +393,34 @@ func (rch *RaceControlHandler) nextSession(w http.ResponseWriter, r *http.Reques
 
 	http.Redirect(w, r, "/live-timing", http.StatusFound)
 }
+
+func (rch *RaceControlHandler) countdown(w http.ResponseWriter, r *http.Request) {
+
+	// broadcast countdown
+	ticker := time.NewTicker(time.Second)
+	i := 4
+
+	for range ticker.C {
+		var countdown string
+
+		i--
+
+		if i > 0 {
+			countdown = strconv.Itoa(i)
+		} else if i == 0 {
+			countdown = "GO"
+		} else {
+			ticker.Stop()
+
+			return
+		}
+
+		err := rch.raceControl.splitAndBroadcastChat(countdown)
+
+		if err != nil {
+			logrus.WithError(err).Error("Unable to broadcast countdown message")
+
+			AddErrorFlash(w, r, "The server was unable to broadcast the countdown message!")
```

Comment:
This error flash will likely only be seen by the user if they refresh the page/load another page. It might be enough to just log the output above and leave it at that - or think of another way of providing a more direct error message to the user